src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/inventory/inventory.component.ts

```diff
@@ -50,23 +49,27 @@ export class InventoryComponent implements OnChanges, OnInit {
       {
         name: this.i18n('Type'),
         prop: 'human_readable_type',
-        flexGrow: 1
+        flexGrow: 1,
+        cellTransformation: CellTemplate.badge,
+        customTemplateConfig: {
+          map: {
+            hdd: { value: 'HDD', class: 'badge-hdd' },
+            'ssd/nvme': { value: 'SSD/NVME', class: 'badge-ssd' }
```

Comment:
```suggestion
            'ssd/nvme': { value: 'SSD/NVMe', class: 'badge-ssd' }
```
Thanks for the eye candy and waiting for #30921.